src/uwuw/uwuw.cpp

```diff
@@ -151,7 +151,6 @@ int UWUW::get_length_of_table(std::string filename, std::string datapath) {
   hid_t arr_space = H5Dget_space(ds);
 
   hsize_t arr_dims[1];
-  int arr_ndim = H5Sget_simple_extent_dims(arr_space, arr_dims, NULL);
```

Comment:
although the return value `arr_ndim` is not used, this call is necessary to populate `arr_dims`